tutorial/Tutorial.md

```diff
@@ -0,0 +1,93 @@
+
+## The Code Example 
+
+In our example, a simple car wash is modelized, where washing coins are used to control the usage of the car wash. Users can register, buy washing coins and use them to start the car wash. For the sake of simplicity, we show here only the function headers of the Move module, the full module can be seen [here](https://github.com/eigerco/pallet-move/blob/main/tests/assets/move-projects/car-wash-example/sources/CarWash.move).
+
+```move
+module DeveloperBob::CarWash {
+    /// Struct stores number of coins for each user.
+    struct Balance has key, store {
+        coins: u8
+    }
+
+    /// Method executes the ICO without money. The module owner (also the car wash owner) gets deposited the minted washing coins.
+    public fun initial_coin_minting(module_owner: &signer) {}
+
+    /// Registers a new user. The account address will be added to the storage Balance with zero initial washing coins.
+    public fun register_new_user(account: &signer) {}
+
+    /// Buys a washing coin for the car wash. Therfore, `COIN_PRICE` will be withdrawn from the user's account.
+    // Note: It would be nice to have the ability to buy multiple coins at once, but it's not implemented for this example.
+    public fun buy_coin(user: &signer) acquires Balance {}
+
+    /// Initiates the washing process by paying one washing coin.
+    public fun wash_car(user: &signer) acquires Balance {}
+}
+```
+
+Each publically usable method will be depicted into an executable script, like this style:
+```move
+script {
+    use DeveloperBob::CarWash;
+    
+    fun initial_coin_minting(account: signer) {
+        CarWash::initial_coin_minting(&account);
+    }
+}
+```
+
+## Publishing Modules
+
+Now we compile the project by using `smove`:
+```sh
+smove build
+```
+
+The compiled bytecode file can be found in the subfolder `build/car-wash-example/bytecode_modules`. In this tutorial we running a Substrate template node with pallet-move already integrated. Description how tu run that can be found [here](https://github.com/eigerco/substrate-node-template-move-vm-test). In polkadot.js via __Developer__->__Extrinsics__ we can select the `moveModule` and extrinsic `publishModule(bytecode, gasLimit)`. Paramter __bytecode__ will be filled by uploading the compiled file `CarWash.mv`. To limit possible fees for the publication, we can adjust the __gasLimit__ or set the value high enough to just accept it. How the needed amount of gas can be estimated properly, will be shown in the [next subsection](#estimation-of-needed-gas).
```

Comment:
Overall suggestion to replace this huge block of text:

The compiled bytecode file `CarWash.mv` can be found at the location:
```
build/car-wash-example/bytecode_modules/CarWash.mv`
```

In polkadot.js: 
- Under the __Developer__ button on top
- Select the __Extrinsics__ option
- In the new __Extrinsics__ page pick the `moveModule` pallet
- For this pallet, choose extrinsic `publishModule(bytecode, gasLimit)`. Parameter explanation:
  - __bytecode__ represents the compiled module bytecode. Fill it by uploading the compiled file `CarWash.mv` (from previous compilation). 
  - __gasLimit__ is used for xxx. Use the optimal value provided by the previous estimation RPC command. Using less than that will make the extrinsic fail, while using more is just not necessary and more expensive)



I would omit this part from here:
~In this tutorial we running a Substrate template node with pallet-move already integrated. Description how tu run that can be found [here](https://github.com/eigerco/substrate-node-template-move-vm-test).~

and mention at the beginning of the tutorial - this is a prerequisite:
 - make sure you run the node with pallet-move already integrated using these [instructions ](https://github.com/eigerco/substrate-node-template-move-vm-test).

Reply:
Done